helm/polaris/tests/poddisruptionbudget_test.yaml

```diff
@@ -0,0 +1,214 @@
+      - equal:
+          path: metadata.name
+          value: polaris-release
+  - it: should set PDB name with override
+    set:
+      podDisruptionBudget.enabled: true
+      nameOverride: polaris-override
+    asserts:
+      - equal:
+          path: metadata.name
+          value: polaris-release-polaris-override
+  - it: should set PDB name with full override
+    set:
+      podDisruptionBudget.enabled: true
+      fullnameOverride: polaris-override
+    asserts:
+      - equal:
+          path: metadata.name
+          value: polaris-override
+
+  # metadata.namespace (with PDB enabled)
+  - it: should set PDB namespace
+    set:
+      podDisruptionBudget.enabled: true
+    asserts:
+      - equal:
+          path: metadata.namespace
+          value: polaris-ns
+
+  # metadata.labels (with PDB enabled)
+  - it: should set PDB default labels
+    set:
+      podDisruptionBudget.enabled: true
+    asserts:
+      - isSubset:
+          path: metadata.labels
+          content:
+            app.kubernetes.io/name: polaris
+            app.kubernetes.io/instance: polaris-release
+            app.kubernetes.io/version: 4.5.6
+            app.kubernetes.io/managed-by: Helm
+            helm.sh/chart: polaris-1.2.3
+
+  # metadata.annotations (with PDB enabled)
+  - it: should not set annotations by default
+    set:
+      podDisruptionBudget.enabled: true
+    asserts:
+      - isNull:
+          path: metadata.annotations
+  - it: should set custom annotations
+    set:
+      podDisruptionBudget.enabled: true
+      podDisruptionBudget.annotations:
+        example.com/policy: "critical"
+        kubernetes.io/description: "PDB for Polaris"
+    asserts:
+      - equal:
+          path: metadata.annotations
+          value:
+            example.com/policy: "critical"
+            kubernetes.io/description: "PDB for Polaris"
+  - it: should template annotations
+    set:
+      podDisruptionBudget.enabled: true
+      podDisruptionBudget.annotations:
+        app.example.com/release: "{{ .Release.Name }}"
+    asserts:
+      - equal:
+          path: metadata.annotations
+          value:
+            app.example.com/release: "polaris-release"
+
+  - it: should set custom maxUnavailable
+    set:
+      podDisruptionBudget.enabled: true
+      podDisruptionBudget.maxUnavailable: 2
+    asserts:
+      - equal:
+          path: spec.maxUnavailable
+          value: 2
+      - isNull:
+          path: spec.minAvailable
+  - it: should set maxUnavailable percentage
+    set:
+      podDisruptionBudget.enabled: true
+      podDisruptionBudget.maxUnavailable: "50%"
+    asserts:
+      - equal:
+          path: spec.maxUnavailable
+          value: "50%"
+      - isNull:
+          path: spec.minAvailable
+
+  # spec.minAvailable
+  - it: should set minAvailable and unset maxUnavailable
+    set:
+      podDisruptionBudget.enabled: true
+      podDisruptionBudget.minAvailable: 2
+      podDisruptionBudget.maxUnavailable: null
```

Comment:
Explicitly unsetting this is not necessary anymore, but it doesn't hurt to keep this line either.